client/state/data-layer/wpcom/jetpack/settings/index.js

```diff
@@ -17,6 +17,11 @@ import {
 	JETPACK_ONBOARDING_SETTINGS_SAVE,
 } from 'state/action-types';
 import { getSiteUrl, getUnconnectedSiteUrl } from 'state/selectors';
+import {
+	normalizeSettings,
+	sanitizeSettings,
+	filterSettingsByActiveModules,
+} from 'state/jetpack/settings/utils';
```

Comment:
If these utilities will be used only for the data layer, maybe we should consider moving them here at some point?

Reply:
Yeah, I plan on doing so in one of the next PRs (where other stuff will be moved/renamed, too). I left them at their current location for now so we wouldn't have to test JP settings for this PR.